qt/scientific_interfaces/Indirect/JumpFitDataPresenter.cpp

```diff
@@ -202,6 +206,19 @@ void JumpFitDataPresenter::addWorkspace(IndirectFittingModel *model,
 void JumpFitDataPresenter::addDataToModel(IAddWorkspaceDialog const *dialog) {
   if (const auto jumpDialog =
           dynamic_cast<JumpFitAddWorkspaceDialog const *>(dialog)) {
+    //  update active data index with correct index based on the workspace name
+    //  and the vector in m_fitDataModel which is in the base class
+    //  indirectFittingModel get table workspace index
+    const auto wsName = jumpDialog->workspaceName().append("_HWHM");
```

Comment:
Not to do with this PR but for multiple selection we could solve the EISF having the wrong name by only doing this appending step in the HWHM case. Propbably better ways but though  would note it here as it occured.